components/layout.js

```diff
@@ -4,7 +4,9 @@ import PropTypes from "prop-types";
 
 const page_wrapper = css`
   margin: 0 auto;
-  max-width: 1600px;
+  max-width: 750px;
```

Comment:
The sticky header bar and the form don't have the same left alignment. I noticed there's still a max-width of 1600 px for the sticky header bar. Maybe there's another 1600px hardcoded value you need to update?

Reply:
yeah, fair enough. Ok, tweaked a bunch of things and now it looks like
<img width="1283" alt="image" src="https://user-images.githubusercontent.com/8228248/65073462-e7c44c00-d960-11e9-9cb4-4b5e38e67173.png">
We should probably change quite a bit of how we do layout, etc. Probably replacing govuk-react will help out a lot. All that custom css 😢